impeller/entity/shaders/blending/blend_select.glsl

```diff
@@ -22,39 +22,51 @@
 // kColor,
 // kLuminosity,
 // Note, this isn't a switch as GLSL ES 1.0 does not support them.
-#define AdvancedBlend(blend_type)           \
```

Comment:
Ah, yeah this shader was totally broken for all backends (impellerc runs the preprocessor), but we just don't use this shader on iOS anymore. GL goldens on CI would have caught this.